src/applications/disability-benefits/2346/components/IntroductionPage.jsx

```diff
@@ -30,26 +29,33 @@ class IntroductionPage extends React.Component {
         <div className="process schemaform-process">
           <ol>
             <li className="process-step list-one">
-              <h3>Prepare</h3>
-              <p>To place an order, you'll need your:</p>
+              <h3 className="vads-u-font-size--h5">Prepare</h3>
+              <p>To place an order, you’ll need your:</p>
               <ul>
                 <li>Shipping address</li>
                 <li>Email address</li>
                 <li>Hearing aid information</li>
               </ul>
               <h4>What if I need help with my order?</h4>
               <p>
-                If you need help ordering hearing aid and betteries and
-                accessories, you cna contact the Devner Logistics Center (DLC)
-                at <a href="tel:303-273-6200">303-273-6200</a>
+                If you need help ordering hearing aid batteries and accessories,
+                you can call the Denver Logistics Center Customer Service
+                Section at
+                <a
+                  aria-label="3 0 3. 2 7 3. 6 2 0 0."
```

Comment:
I'm not sure if this is recommended to add at this time? [ref](https://dsva.slack.com/archives/C8E985R32/p1580485449008300)

Reply:
Done on. both numbers